server/src/main/java/org/opensearch/index/IndexingPressure.java

```diff
@@ -38,29 +38,28 @@
 
     private static final Logger logger = LogManager.getLogger(IndexingPressure.class);
 
-    private final AtomicLong currentCombinedCoordinatingAndPrimaryBytes = new AtomicLong(0);
-    private final AtomicLong currentCoordinatingBytes = new AtomicLong(0);
-    private final AtomicLong currentPrimaryBytes = new AtomicLong(0);
-    private final AtomicLong currentReplicaBytes = new AtomicLong(0);
+    final AtomicLong currentCombinedCoordinatingAndPrimaryBytes = new AtomicLong(0);
+    final AtomicLong currentCoordinatingBytes = new AtomicLong(0);
+    final AtomicLong currentPrimaryBytes = new AtomicLong(0);
+    final AtomicLong currentReplicaBytes = new AtomicLong(0);
```

Comment:
I guess the intent is to make it visible to the subclass... If yes make this `protected`

Reply:
intent was also the to keep this as package private. Agree there are tradeoffs based on which instance is created ie ``IndexingPressure`` vs ``ShardIndexingPressure``